web/js/map-OpenLayers.js

```diff
@@ -115,6 +115,29 @@ var fixmystreet = fixmystreet || {};
         fixmystreet.markers.redraw();
       },
 
+      reorder_visible: function() {
+          var visible = ''
+          var $itemList = $('.item-list');
+          var $visibleReports = $('.visible-reports');
+          for (var i = 0; i < fixmystreet.markers.features.length; i++) {
```

Comment:
I assume marker order here is the same as sidebar list order, might be worth a comment confirming that.